DhsTranslatorTest/src/test/java/edu/gemini/dhs/translator/test/DhsTranslatorTest.java

```diff
@@ -246,6 +256,43 @@ public void testSetKeywordWrongType() throws ClientProtocolException,
         assertEquals("Operation did not fail.", "error", status.asText());
     }
 
+    @Test
+    public void testConcurrentCalls() throws InterruptedException,
+            ExecutionException {
+        final ConnectionRequester connectionRequester = new ConnectionRequester();
+
+        Callable<String> task = new Callable<String>() {
```

Comment:
This is just a test case but still a good habit that we've adopted is to mark everything `final` that can be final.  The absence of the `final` modifier then becomes a warning that something tricky is going to happen.